NEWS.md

```diff
@@ -10,7 +10,6 @@
   format specifiers (#472, @russHyde)
 * New style SNAKE_CASE for `object_name_linter()` (#494, @AshesITR)
 * RStudio source markers are cleared when there are no lints (#520, @AshesITR)
-* New `assignment_spaces()` lintr. (#538, @f-ritter)
```

Comment:
Not sure what to do about attribution here...

Reply:
I had the same hesitation... I'll mark down in the follow-up to be sure to thank them.